hayabusa/builtin/System/Sys_7045_Critical_LateralMovement-SMBEXEC.yml

```
@@ -1,12 +1,12 @@
 author: longmd - n0n$la$
 date: 2023/10/16
-modified: 2022/10/16
+modified: 2023/10/19
 
 title: SMBExec Lateral Movement
-details: 'Svc: %ServiceName% ¦ Path: %ImagePath% ¦ Acct: %AccountName% ¦ StartType: %StartType%'
+details: 'Svc %ServiceName% was installed (%StartType%) by %AccountName%'
```

```suggestion
details: 'Svc: %ServiceName% ¦ Path: %ImagePath% ¦ Acct: %AccountName% ¦ StartType: %StartType%'
```

When `json-timeline` option was used  in Hayabusa, details data split by `¦` to check easily, So I think this format suggest.